recipe/meta.yaml

```diff
@@ -10,11 +10,11 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx or win]
+  skip: true  # [osx]
 
 requirements:
   run:
-    - cuda-nsight 12.0.78               # [not aarch64]
+    - cuda-nsight 12.0.78               # [linux64 or ppc64le]
```

Comment:
Is there a reason to change this?

Reply:
Yes cuda-nsight is not supported on Windows so it needs some change
https://github.com/conda-forge/cuda-nsight-feedstock/blob/main/recipe/meta.yaml#L14-L15

Reply:
How about?

```suggestion
    - cuda-nsight 12.0.78               # [not (aarch64 or win)]
```